src/EnergyPlus/OutputProcessor.cc

```diff
@@ -582,18 +470,20 @@ namespace OutputProcessor {
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
 
         ReportingFrequency ReportFreq(ReportingFrequency::Hourly); // Default
+        // TODO: I think it's supposed to be upper case already, but tests aren't doing that at least...
+        const std::string FreqStringUpper = UtilityRoutines::MakeUPPERCase(FreqString);
```

Comment:
I wasn't sure if this was really true (and then I should just modify the tests to be upper case as well) or not.